appserver/tests/quicklook/QuickLook_Test_Instructions.html

```diff
@@ -237,8 +237,8 @@ <H1 STYLE="margin-left: 0.83in"><FONT FACE="Trebuchet MS, sans-serif"><FONT SIZE
 deploy runtest&quot;</P>
 <P STYLE="margin-bottom: 0in">Since QL runs in multiple profiles, the
 test cases need to be defined in the 3 test lists (under
-quicklook/testng dir) for the profiles: testng_wd.xml for the Web
-profile, testng_gd.xml for Glassfish profile, testng_debug.xml for
+quicklook/testng dir) for the profiles: testng_web_profile.xml for the Web
+profile, ttestng_full_profile.xml for Glassfish full profile, testng_debug.xml for
```

Comment:
extra `t` in `ttestng_full_profile.xml`?

Reply:
> ttestng_full_profile.xml

That's an amazing eagle eyed spot!